app/src/main/java/edu/uc/zhukv/droneradarmap/service/AirportService.kt

```diff
@@ -10,7 +10,7 @@ import retrofit2.Callback
 import retrofit2.Response
 
 class AirportService {
-    fun fetchAirports(): MutableLiveData<ArrayList<Airport>> {
+    internal fun fetchAirports(): MutableLiveData<ArrayList<Airport>> {
```

Comment:
Changed visibility modifier to internal
https://kotlinlang.org/docs/visibility-modifiers.html#classes-and-interfaces